NEWS.md

```diff
@@ -6,6 +6,10 @@ Main
 
 ### ![][badge-🐛bugfix] Bug fixes
 
+- Update RRTMGP to v0.19.1, which fixes the sea salt aerosol lookup table.
+  Sea salt aerosol is added to the target amip config. 
+  PR [3374](https://github.com/CliMA/ClimaAtmos.jl/pull/3374)
```

Comment:
Being pedantic here: the update of RRTMGP is only in the enviroments, not in ClimaAtmos package itself. (No need to change anything)